server/webapp/WEB-INF/rails/config/routes.rb

```diff
@@ -187,19 +187,22 @@
     get "templates/:name" => "config_view/templates#show", as: :config_view_templates_show, constraints: {name: TEMPLATE_NAME_FORMAT}
   end
 
-  scope 'environments' do
+  get 'environments', to: redirect('admin/environments'), as: :environment_redirect
```

Comment:
Currently `/go/environments` is redirected to => `/go/admin/environments`.

Need to consider redirects for the following routes -

- `go/environments/foo/show` =>`go/admin/environments/foo/show`
- `go/environments/new` => `go/admin/environments/new`
- POST  to `/go/environments/:env_name` => `/go/admin/environments/:env_name`

Reply:
This is to make sure that old links for env page will continue to work. We will put deprecation of old routes in the release note.